src/test/unit/controls/Bar/Bar-spec.js

```diff
@@ -70,7 +70,7 @@ describe("Bar", () => {
         });
         it("throws error when no values are provided", () => {
             const input = utils.deepClone(getInput(valuesDefault));
-            input.values = [];
+            input.values = undefined;
```

Comment:
Can you add one more test to the specs to `Does not throw error when empty values array is provided`?